sh/install.sh

```diff
@@ -76,13 +95,13 @@ function install_python() {
     if [ $OPERATING_SYSTEM == "DEBIAN" ]; then
         echo "Installing Python using apt-get"
         apt-get update >> ${INSTALL_LOG} 2>&1
-        apt-get install python -y >> ${INSTALL_LOG} 2>&1
+        apt-get install python3 -y >> ${INSTALL_LOG} 2>&1
```

Comment:
```suggestion
        apt-get install python3 python3-pip -y >> ${INSTALL_LOG} 2>&1
```
as if we know python wasn't installed, then pip wouldn't be there either, so why not install both